lxd/storage/backend_lxd.go

```diff
@@ -3323,8 +3323,15 @@ func (b *lxdBackend) SetInstanceQuota(inst instance.Instance, size string, vmSta
 		return err
 	}
 
-	// Apply the main volume quota.
+	// If no size is provided, block volumes should get resized to the pool's default size for block volumes.
```

Comment:
@hamistao im not sure we want this behavior, lets discuss before going forward with this.